Bixal/move.mil
# Parser manager

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

Hooks up the file upload form to the functionality of the command.

## Testing

1. Pull changes
2. Install module.
3. Browse to http://move.mil.localhost:8000/admin/config/parser/manager
4. Open up a database shell.
5. Try uploading files and check the tables in the database.
6. Try truncating files and check the tables in the database.
## Screenshots

_Attach relevant before and after screenshots here._
